docs/api/tutorials_intro.md

```diff
@@ -238,6 +237,6 @@ A [tutorial for creating and using a bootloader](../tutorials/bootloader.html),
 </tbody>
 </table>
 
-## Migrating to Mbed OS 5
+## Mbed OS 2 to bare metal
 
-[Mbed OS 2 to 5 migration tutorial](../tutorials/migrating-to-mbed-os-5.html)
+If you're moving from Mbed OS 2, see [the Mbed OS 6 bare metal porting guide](../porting-a-target-from-mbed-os-2-to-mbed-os-6-bare-metal.html).
```

Comment:
```suggestion
If you're moving from Mbed 2, see [the Mbed OS 6 bare metal porting guide](../porting-a-target-from-mbed-os-2-to-mbed-os-6-bare-metal.html).
```

Reply:
Rejecting this purely because I'm going to make it fit both 2>bare metal and 5>6 API deprecation.